src/styles/index.scss

```diff
@@ -1,3 +1,220 @@
+    margin-left: 20px;
+    &:first-child {
+      margin: 0;
+    }
+    & > a {
+      font-size: 12px;
+      font-weight: 500;
+      height: var(--header-height);
+      align-content: center;
+      color: #060b35;
+      transition: 0.3s;
+      &:hover {
+        color: var(--link-color);
+      }
+    }
+  }
+}
+
+.is-active {
+  color: var(--link-color) !important;
+  position: relative;
+}
+
+a.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  background-color: var(--link-color);
+  left: 0;
+  width: 100%;
+  border-radius: 5px;
+  height: 4px;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+  transition: 0.3s;
+
+  &__imgBlock {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+    & > img {
+      height: 134px;
+      width: 160px;
+    }
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #060b35;
+  }
+  &__productCode {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+    color: #616070;
+  }
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+    & > div {
+      font-size: 10px;
+      line-height: 14px;
+      height: 16px;
+      font-weight: 400;
+      display: flex;
+      align-items: center;
+      text-align: right;
+    }
+  }
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: 16px;
+    text-align: center;
+    width: 100%;
+    color: #fff;
+    border: none;
+    box-sizing: border-box;
+    height: 40px;
+    border-radius: 5px;
+    background-color: #00acdc;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    transition: 0.3s;
+    cursor: pointer;
+    &:hover {
+      background-color: #fff;
+      color: #00acdc;
+      border: 1px solid #00acdc;
+    }
+  }
+  &:hover {
+    transform: scale(1.2);
+  }
+  &:hover .card__title {
+    color: #34568b;
+  }
+}
+
+.priceBlock {
+  margin-top: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  &__label {
+    color: #616070;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+  &__price {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: #060b35;
+  }
+}
+
+.stars-4 {
+  display: flex;
+  & > div {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+    &:not(:first-child) {
+      margin-left: 4px;
+    }
+    &:nth-last-child(n + 2) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+}
+
+main {
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 46px 48px;
+  padding: 50px 40px;
+  justify-content: center;
+}
```

Comment:
The 'main' element is correctly styled to use a grid layout with the specified column count and gap, as required by the task.